modules/SFM/SFM.h

```diff
@@ -34,20 +34,23 @@ CopyPolicy: Released under the terms of the GNU GPL v2.0.
 The module uses a complete Structure From Motion (SFM) pipeline 
 for the computation of the extrinsics parameters between two 
 different views. These parameters are then used to rectify the 
-images and to compute a depth map using the H. Hirschmuller 
-Algorithm (CVPR 2006) implemented since Opencv 2.2. The 
-Kinematics of the iCub is used to guess the current camera 
-positions, then visual features are used to refine this model. 
+images and to compute a depth map using either the H. Hirschmuller 
+Algorithm (CVPR 2006), implemented since Opencv 2.2, or the ELAS library. 
+The Kinematics of the iCub is used to guess the current camera positions, 
+then visual features are used to refine this model.
 Before starting, make sure you have calibrated the intrinsics 
 parameters. For the stereo calibration see the module <a 
 href="http://wiki.icub.org/iCub/main/dox/html/group__icub__stereoCalib.html">stereoCalib</a>. 
-The module provides three output ports: the first one is the 
+The module provides five output ports: the first one is the 
```

Comment:
I said doc but I meant the `SFM.h` file, just a line pointing out the new option is more than enough.